### PR TITLE
IA-1467 Quote command to handle workspace names with spaces

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-1467

The fix:
```
export WORKSPACE_NAME="$(basename $(dirname "$PWD"))"
```
needs to be 
```
export WORKSPACE_NAME="$(basename "$(dirname "$PWD")")"
```

I also added an auto test which tests the EVs in a directory with spaces.

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/jupyter/init-actions.sh [hunk 1]
```diff
@@ -372,7 +372,7 @@ if [[ "${ROLE}" == 'Master' ]]; then
       # fix for https://broadworkbench.atlassian.net/browse/IA-1453
       # TODO: remove this when we stop supporting the legacy docker image
       if [ ! -z ${WELDER_DOCKER_IMAGE} ] && [ "${WELDER_ENABLED}" == "true" ] ; then
-        retry 3 docker exec -u root ${JUPYTER_SERVER_NAME} sed -i -e 's/"$(basename "$PWD")"/"$(basename $(dirname "$PWD"))"/' ${JUPYTER_HOME}/scripts/kernel/kernel_bootstrap.sh
+        retry 3 docker exec -u root ${JUPYTER_SERVER_NAME} sed -i -e 's/export WORKSPACE_NAME=.*/export WORKSPACE_NAME="$(basename "$(dirname "$PWD")")"/' ${JUPYTER_HOME}/scripts/kernel/kernel_bootstrap.sh
```
Comment (Qi77Qi) [inline http/src/main/resources/jupyter/init-actions.sh]: there's 2 export here

Comment (rtitle) [reply http/src/main/resources/jupyter/init-actions.sh]: it's 2 arguments to `sed` unless I'm missing something..
```
s/export WORKSPACE_NAME=.*/export WORKSPACE_NAME="…"/
```

Comment (Qi77Qi) [reply http/src/main/resources/jupyter/init-actions.sh]: oh..my bad

**Comment by rtitle** [comment]
Also made https://github.com/DataBiosphere/terra-docker/pull/50/

**Review by akarukappadath — APPROVED** [review]
thanks for taking care of this :)

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**